addon/index.js

```diff
@@ -71,13 +72,26 @@ function updatePrefs() {
   Object.assign(settings, {
     BASE_URL: env.BASE_URL,
     ALLOWED_ORIGINS: env.BASE_URL + '/*',
+    ALLOWED_ORIGINS_READONLY: 'https://www.mozilla.org/*,about:home',
```

Comment:
Rather than being hardcoded here, I think these origins should appear [as a default preference in `package.json`](https://github.com/ckprice/testpilot/blob/issue-463-content-detection/addon/package.json#L37) and get picked up [from `simplePrefs.prefs`](https://github.com/ckprice/testpilot/blob/issue-463-content-detection/addon/index.js#L67). Then, we can change them without rebuilding the addon - e.g. for QA & integration testing.